main.py

```diff
@@ -270,6 +276,13 @@ async def execute_opportunities_webhook(webhook_url, job_message, internship_mes
 
 
 async def main():
+    # Creates table in database
+    command_line_value = utils.extract_command_value()["create_table"]
```

Comment:
Use a more specific variable name for the command line value. Such as `with_create_table`.

Reply:
Resolved [here](https://github.com/acmcsufoss/acmOpportunities/pull/19/commits/41e95dfe43d3122a24bffb772f72ecce05315faf)